homework/add-even/addEven.hpp

```diff
@@ -1,8 +1,13 @@
 #pragma once
+#include <iostream>
 #include <vector>
 
-int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+int addEven(const std::vector<int> &numbers) {
+  auto r{0};
```

Comment:
What is 'r'? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...